launch/launch/launch_service.py

```diff
@@ -341,6 +342,11 @@ def _on_sigquit(signum, frame):
                     pass
                 except asyncio.CancelledError:
                     _logger.error('asyncio run loop was canceled')
+                except Exception as exc:
+                    msg = 'Caught exception in launch (see debug for traceback): {}'.format(exc)
+                    _logger.error(msg)
+                    _logger.debug(traceback.format_exc())
```

Comment:
Oops, I missed the `error` call right above this line. Nevermind.

Reply:
Right now debug doesn't go to a file, but perhaps it will in the future (perhaps related https://github.com/ros2/launch/issues/104).

Do you think it's good enough to have the traceback in debug until that happens?

Reply:
> Do you think it's good enough to have the traceback in debug until that happens?

I do.